arm/Microsoft.RecoveryServices/vaults/deploy.bicep

```diff
@@ -126,43 +126,34 @@ resource rsv 'Microsoft.RecoveryServices/vaults@2021-08-01' = {
   properties: {}
 }
 
-module rsv_backupStorageConfig 'backupStorageConfig/deploy.bicep' = {
+module backupStorageConfiguration 'backupStorageConfig/deploy.bicep' = {
```

Comment:
Please change back to rsv_backupStorageConfig. Naming for modules called from a parent resource should be like
`<nameOfParentResource>_<childResourceType>` in the parent. in this case `<nameOfParentResource> = rsv` and `<childResourceType> = backupStorageConfig`.

In the child template you did correctly, no need to change back.

Reply:
Done!